src/AvBaseInput.js

```diff
@@ -149,6 +157,9 @@ export default class AvBaseInput extends Component {
     if (this.props.type === 'checkbox') {
       return event.target.checked ? this.props.trueValue : this.props.falseValue;
     }
+    if (this.props.type === 'select' && this.props.multiple){
+      return [...event.target.options].filter(({selected}) => selected).map(({value}) => value);
```

Comment:
You probably don't need the spread here; both filter and map will return new arrays themselves.